rpc_jobs/rpc_openstack.yml

```diff
@@ -437,6 +441,65 @@
     jobs:
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
 
+- project:
+    name: 'rpc-openstack-master-mnaio-release'
+    repo_name: 'rpc-openstack'
+    repo_url: 'https://github.com/rcbops/rpc-openstack'
+    branch: "master"
```

Comment:
The rpc-openstack jobs to validate a release should actually come from the -rc branches, not the mainline branches. I guess there may actually be a requirement to run both the mainline and -rc branch jobs as the mainline becomes the next rc. Will this automagically figure out which right jobs are, or does this have to be more specific?

Reply:
The release jobs actually build w/ the version being proposed, so the `branch` listed in these jobs is really just to distinguish them from one-another. I can certainly change them to `master-rc` and `newton-rc` if that makes it more clear (and I suppose it should, since that's where the proposed changes are coming from).

Reply:
Changed the branch to `-rc` to better reflect the branch the change we're running with is from.